shipshape.yml

```diff
@@ -41,6 +41,44 @@ checks:
             - import configuration
             - synchronize configuration
             - use PHP for google analytics tracking visibility
+    - name: '[FILE] Disallowed permissions for anonymous role'
+      severity: high
```

Comment:
```suggestion
```

When introducing new checks we should add them without the severity — this will allow us to warn in CI before outright blocking.